ansible/roles/sonic-common/tasks/main.yml

```diff
@@ -18,6 +24,14 @@
 - include: aptrepo.yml
   tags: repo
 
+# Dbus
+- name: Install dbus
+  become: true
+  apt: pkg=dbus
```

Comment:
dbus is required by systemd. Not sure about all platform, we have **when** condition here:
when: sonic_hwsku == "AS7512". On our pure ONIE image(not configured by ansible) dbus was required by systemctl.